### PR TITLE
Bug 1824358: null creationTimestamp shows as relative time of 50 years ago

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @TheRealJon
/cc @benjaminapetersen @spadgett

### CODE REVIEW

File: frontend/public/components/utils/datetime.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@ import * as _ from 'lodash-es';
 
 // Behaves like moment.js's fromNow
 export const fromNow = (dateTime, now = undefined, options = { omitSuffix: false }) => {
+  // Check for null. If dateTime is null, it returns incorrect date and time of Wed Dec 31 1969 19:00:00 GMT-0500 (Eastern Standard Time)
+  if (!dateTime) {
+    return 'N/A';
```
Comment (spadgett) [inline frontend/public/components/utils/datetime.ts:7]: We usually show `-` for a missing value.

**Comment by cyril-ui-developer** [comment]
> Can you add a unit test to the existing `fromNow` tests?

@spadgett Sure, will work on it. Can I add the unit test to this PR?

**Comment by spadgett** [comment]
Yes, same PR. It should just be an additional test here:

https://github.com/openshift/console/blob/master/frontend/__tests__/components/utils/datetime.spec.ts#L9

File: frontend/__tests__/components/utils/datetime.spec.ts [hunk 2]
```diff
@@ -89,6 +89,10 @@ describe('fromNow', () => {
     expect(fromNow(new Date('Feb 02 1971'), new Date('Jan 01 1970'))).toEqual('a year from now');
     expect(fromNow(new Date('Feb 02 1973'), new Date('Jan 01 1970'))).toEqual('3 years from now');
   });
+
```
Comment (benjaminapetersen) [inline frontend/__tests__/components/utils/datetime.spec.ts:92]: Thx for the test!

**Pull request merged**